01-Login/README.md

```diff
@@ -542,9 +514,9 @@ import React, { Component } from "react";
 
 export default function withAuthentication(WrappedComponent, auth0) {
   return class WithAuthentication extends Component {
-    state = { loading: true };
+    state = { loading: true, isAuthenticated: false };
 
-    async componentDidMount() {
+    async componentWillMount() {
```

Comment:
@elkdanger What reason behind the change? according to React's documentation, this will be deprecated and it shouldn't be used anymore: https://reactjs.org/docs/react-component.html#unsafe_componentwillmount

Reply:
@glrodasz Oops, that's a mistake. I think I was trying to fix something out and forgot to change it back. I'll fix that :)